SUPPORT.md

```diff
@@ -31,8 +31,10 @@ Further information, including the MSRC PGP key, can be found in the [Security T
 
 ## Microsoft Support Policy
 
-The Azure SDK libraries for JavaScript will not be guaranteed to work on Node.js versions that have reached their end of life. Dropping support for such Node.js versions may be done without increasing the major version of the Azure SDK libraries.
+The Azure SDK libraries for JavaScript will not be guaranteed to work on Node.js versions that have reached their end of life. Dropping support for such Node.js versions may be done without increasing the major version of the Azure SDK libraries.  
 
 We strongly recommend migration to [LTS versions of Node.js](https://github.com/nodejs/release#release-schedule) to be eligible for technical support. In terms of browsers, we support latest versions of Safari, Chrome, Edge and Firefox. We will make reasonable efforts to support the Azure SDK libraries on Node.js that have reached their end of life less than six months prior, unless security or other considerations require otherwise.
 
+The Azure SDK for JavaScript is written in TypeScript.  To take full advantage of TypeScript, we use the latest version of TypeScript available.  We recommend using version 4.x+.
```

Comment:
```suggestion
The Azure SDK for JavaScript is written in TypeScript.  To take full advantage of TypeScript, we write in the latest version of TypeScript available.  We will test and support minor versions released in the last two years to match the support window of Definitely Typed, the community driven source for typings that are not provided by package authors: https://github.com/DefinitelyTyped/DefinitelyTyped#support-window
```

Reply:
Here's another thought along the same lines

>  we support and test with the minor versions released in the last two years, in order to ensure that our code is up-to-date and compatible with the latest features and improvements. This support window aligns with the support windows of Definitely Typed, a popular TypeScript type definitions repository, and reflects our dedication to staying current with the latest developments in the TypeScript ecosystem.

Reply:
nit: DefinitelyTyped is one word

Reply:
I might even link to DT's support-window SVG file.